### ISSUE TITLE
Move out static content out from provider classes

### ISSUE THREAD

**Issue by laxitshahi** [llm-proxy/llm-proxy#109]
There is a lot of static data within each provider's (cohere, openai...) class. Move this data out to the yml_configuration file and use imports to access it.

- [x] Cost Routing
- [ ] Category Routing

Example of in class data:
![Image](https://github.com/llm-proxy/llm-proxy/assets/46412618/4bd559c0-36eb-4c8e-8b65-ef0a474d6d4f)

Depends on #94

**Comment by laxitshahi** [comment]
Internal Configuration will now just be a simple python file with a dictionary that contains all of the cost and category information

**Comment by laxitshahi** [comment]
Task transfered to @dinuja9

**Comment by laxitshahi** [comment]
Cost

**Comment by laxitshahi** [comment]
Cost data moved to internal config, but category routing remains as we may put that on hold in favour of proficiency routing
